src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,66 @@
+package mate.academy.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String email;
```

Comment:
I think we need to use this annotation with email @Column(unique = true), because the email must be unique

Reply:
Agree with @vika-trach, let's do it.